lib/model/model_test.go

```diff
@@ -1218,6 +1225,113 @@ func TestAutoAcceptFallsBackToID(t *testing.T) {
 	}
 }
 
+func TestAutoAcceptPausedWhenFolderConfigChanged(t *testing.T) {
+	// Existing folder
+	id := srand.String(8)
+	idOther := srand.String(8) // To check that path does not get changed.
+	defer os.RemoveAll(filepath.Join("testdata", id))
```

Comment:
The `idOther` needs removing too. Same for existing `TestAutoAcceptExistingFolder` test. My testdata is littered with these random dirs ;)  
Or run all tests in temporary dirs anyway.

Reply:
Uh yeah, good stop.